pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -502,6 +553,7 @@ func (f *framework) RunPermitPlugins(
 				klog.Error(msg)
 				return NewStatus(Error, msg)
 			}
+			klog.V(10).Infof("wait complete for permit plugins for pod %q: %v", pod.Name, statusCode)
```

Comment:
nit: let's also log the permit plugin name. 
nit: no need to log statusCode (it's Wait).

Reply:
There is no specified or explicit permit plugin here, here is the wait for all the permit plugin to complete.

Reply:
done